### PR TITLE
Add grid sorting styling

### PR DESCRIPTION
**Pull request by mmart1n**
Related to #1275

### CODE REVIEW

File: en/components/grids_templates/sorting.md [hunk 1]
```diff
@@ -124,6 +124,127 @@ You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad
...
+`igx-palette` generates a color palette based on the primary and secondary colors that are passed:
+
+```scss
+$primary-color: #09f;
+$secondary-color: #e41c77;
+
+$light-palette: igx-palette($primary: $primary-color, $secondary: $secondary-color);
+```
+
+And then with [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) we can easily retrieve color from the pallete.
```
Comment (HristoP96) [inline en/components/grids_templates/sorting.md]: **palette**, not `pallete`.  Also add a code snippet to demonstrate how the colors are retrieved from the palette

File: en/components/grids_templates/sorting.md [hunk 2]
```diff
@@ -124,6 +124,127 @@ You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad
...
+
+Instead of hardcoding the color values like we just did, we can achieve greater flexibility in terms of colors by using the [`igx-palette`]({environment:sassApiUrl}/index.html#function-igx-palette) and [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) functions.
+
+`igx-palette` generates a color palette based on the primary and secondary colors that are passed:
+
+```scss
+$primary-color: #09f;
+$secondary-color: #e41c77;
+
+$light-palette: igx-palette($primary: $primary-color, $secondary: $secondary-color);
```
Comment (HristoP96) [inline en/components/grids_templates/sorting.md]: I think `$custom-palette` wil be a better name (adds more genericity).

File: en/components/grids_templates/sorting.md [hunk 3]
```diff
@@ -124,6 +124,127 @@ You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad
...
+
+```scss
+// Extending the global light-schema
+$my-custom-schema: extend($light-schema, 
+    (
+        igx-grid: $custom-grid-schema
+    )
+);
+
+// Defining our custom theme with the global light schema
```
Comment (HristoP96) [inline en/components/grids_templates/sorting.md]: You define it with your custom schema, not the `global light`

File: en/components/grids_templates/sorting.md [hunk 4]
```diff
@@ -124,6 +124,127 @@ You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad
...
+Instead of hardcoding the color values like we just did, we can achieve greater flexibility in terms of colors by using the [`igx-palette`]({environment:sassApiUrl}/index.html#function-igx-palette) and [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) functions.
+
+`igx-palette` generates a color palette based on the primary and secondary colors that are passed:
+
+```scss
+$primary-color: #09f;
+$secondary-color: #e41c77;
+
+$light-palette: igx-palette($primary: $primary-color, $secondary: $secondary-color);
+```
```
Comment (HristoP96) [inline en/components/grids_templates/sorting.md]: Also, the colors are the same as the `default palette`, it will be better if you change them.

**Comment by mmart1n** [comment]
@HristoP96 requested changes done

**Comment by HristoP96** [comment]
Also could you add this section in the JP version

File: en/components/grids_templates/sorting.md [hunk 5]
```diff
@@ -124,6 +124,132 @@ You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad
...
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-sorting-styling-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+}
+@@if (igxName === 'IgxTreeGrid') {
+#### Demo
+
+<div class="sample-container loading" style="height:550px">
+    <iframe id="tree-grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/tree-grid/treegrid-sorting-styling' width="100%" height="100%" seamless frameBorder="0" class="lazyload"></iframe>
```
Comment (HristoP96) [inline en/components/grids_templates/sorting.md]:
```suggestion
    <iframe id="tree-grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/tree-grid/treegrid-sorting-styling' width="100%" height="100%" seamless frameBorder="0" class="lazyload no-theming"></iframe>
```

File: en/components/grids_templates/sorting.md [hunk 6]
```diff
@@ -124,6 +124,132 @@ You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad
...
+```
+
+Don't forget to include the themes in the same way as it was demonstrated above.
+
+@@if (igxName === 'IgxGrid') {
+#### Demo
+
+<div class="sample-container loading" style="height:550px">
+    <iframe id="grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/grid/grid-sorting-styling' width="100%" height="100%" 
+        seamless frameBorder="0" class="lazyload"></iframe>
```
Comment (HristoP96) [inline en/components/grids_templates/sorting.md]:
```suggestion
        seamless frameBorder="0" class="lazyload no-theming"></iframe>
```

File: en/components/grids_templates/sorting.md [hunk 7]
```diff
@@ -124,6 +124,132 @@ You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad
...
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="tree-grid-sorting-styling-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+}
+@@if (igxName === 'IgxHierarchicalGrid') {
+#### Demo
+
+<div class="sample-container loading" style="height:510px">
+    <iframe id="hierarchical-grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-sorting-styling' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
```
Comment (HristoP96) [inline en/components/grids_templates/sorting.md]:
```suggestion
    <iframe id="hierarchical-grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-sorting-styling' width="100%" height="100%" seamless frameBorder="0" class="lazyload no-theming"></iframe>
```

File: jp/components/grids_templates/sorting.md [hunk 8]
```diff
@@ -123,6 +123,132 @@ public ngOnInit() {
...
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-sorting-styling-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+}
+@@if (igxName === 'IgxTreeGrid') {
+#### Demo
+
+<div class="sample-container loading" style="height:550px">
+    <iframe id="tree-grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/tree-grid/treegrid-sorting-styling' width="100%" height="100%" seamless frameBorder="0" class="lazyload"></iframe>
```
Comment (HristoP96) [inline jp/components/grids_templates/sorting.md]:
```suggestion
    <iframe id="tree-grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/tree-grid/treegrid-sorting-styling' width="100%" height="100%" seamless frameBorder="0" class="lazyload no-theming"></iframe>
```

File: jp/components/grids_templates/sorting.md [hunk 9]
```diff
@@ -123,6 +123,132 @@ public ngOnInit() {
...
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="tree-grid-sorting-styling-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+}
+@@if (igxName === 'IgxHierarchicalGrid') {
+#### Demo
+
+<div class="sample-container loading" style="height:510px">
+    <iframe id="hierarchical-grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-sorting-styling' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
```
Comment (HristoP96) [inline jp/components/grids_templates/sorting.md]:
```suggestion
    <iframe id="hierarchical-grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-sorting-styling' width="100%" height="100%" seamless frameBorder="0" class="lazyload no-theming"></iframe>
```

File: jp/components/grids_templates/sorting.md [hunk 10]
```diff
@@ -123,6 +123,132 @@ public ngOnInit() {
...
+```
+
+Don't forget to include the themes in the same way as it was demonstrated above.
+
+@@if (igxName === 'IgxGrid') {
+#### Demo
+
+<div class="sample-container loading" style="height:550px">
+    <iframe id="grid-sorting-styling-iframe" src='{environment:demosBaseUrl}/grid/grid-sorting-styling' width="100%" height="100%" 
+        seamless frameBorder="0" class="lazyload"></iframe>
```
Comment (HristoP96) [inline jp/components/grids_templates/sorting.md]:
```suggestion
        seamless frameBorder="0" class="lazyload no-theming"></iframe>
```

**Pull request merged**